test/Sql/SelectTest.php

```diff
@@ -1397,4 +1399,21 @@ public function providerData()
             [$select52, $sqlPrep52, [],    $sqlStr52, $internalTests52],
         ];
     }
+
+    public function testPropagateAdapterWithGetSqlStringThatAlreadyBroughtBySqlObject()
+    {
+        if (extension_loaded('mysqli')) {
```

Comment:
I personally think that will way too much, we are here make sure as sample, the "Adapter" behaviour is brought when it already passed to Sql object.